=== command
CI="true" GITLAB_CI="true" CI_PROJECT_PATH="project_name/project_name" CI_PIPELINE_SOURCE="merge_request_event" CI_MERGE_REQUEST_TARGET_BRANCH_NAME="main" CI_MERGE_REQUEST_PROJECT_URL="https://some.project.url.test.placeholder" CI_JOB_TOKEN="some-token-test-placeholder" CI_COMMIT_REF_NAME="some/branch-name" CI_COMMIT_SHA="<MASKED>" CI_PROJECT_URL="https://example.com/gitlab-org/gitlab-foss" CI_JOB_URL="https://gitlab.com/gitlab-examples/ci-debug-trace/-/jobs/379424655" CI_MERGE_REQUEST_IID="unused-iid-test-placeholder" CI_MERGE_REQUEST_DIFF_BASE_SHA="unused-commit-test-placeholder" CI_MERGE_REQUEST_TITLE="unused-merge-request-title-test-placeholder" SEMGREP_APP_TOKEN="fake-key-from-tests" SEMGREP_USER_AGENT_APPEND="pytest" SEMGREP_SETTINGS_FILE="<MASKED>" SEMGREP_VERSION_CACHE_PATH="<MASKED>" SEMGREP_ENABLE_VERSION_CHECK="0" SEMGREP_SEND_METRICS="off" semgrep ci --no-suppress-errors
=== end of command

=== exit code
2
=== end of exit code

=== stdout - plain

=== end of stdout - plain

=== stderr - plain
ci() got an unexpected keyword argument 'matching_explanations'
Traceback (most recent call last):
  File "/home/runner/work/semgrep/semgrep/cli/src/semgrep/commands/wrapper.py", line 35, in wrapper
    func(*args, **kwargs)
TypeError: ci() got an unexpected keyword argument 'matching_explanations'

=== end of stderr - plain